dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ set(MODELICA_PREASSEMBLED_MODELS
   EventQuadripoleDisconnection.xml
   EventSetPointBoolean.xml
   EventSetPointReal.xml
+  EventSetPointDouble.xml
```

Comment:
There is probably an issue of merge here

Reply:
It seems that this preassembled was not added in the list so Marco added it here